lib/logitech_receiver/settings_templates.py

```diff
@@ -250,6 +256,15 @@ def _feature_adjustable_dpi():
 					label=_DPI[1], description=_DPI[2],
 					device_kind=_DK.mouse)
 
+def _feature_pointer_speed():
+	"""Pointer Speed feature"""
+	# min and max values taken from usb traces of Win software
+	return feature_range(_POINTER_SPEED[0], _F.POINTER_SPEED, 0x002e, 0x01ff,
+					read_function_id=0x0,
+					write_function_id=0x10,
+					bytes_count=2,
+					label=_POINTER_SPEED[1], description=_POINTER_SPEED[2],
+					device_kind=_DK.mouse)
```

Comment:
Hey, based on these docs (https://lekensteyn.nl/files/logitech/x2205_pointermotionscaling.html), wouldn't a range from 0x0010 to 0x1000 be better? (What happens if you try those settings?)